ramls/loan.json

```diff
@@ -103,6 +103,11 @@
       "type": "string",
       "format": "date-time"
     },
+    "markedMissingDate": {
```

Comment:
Hi @Svitlana-Zmiivska, from what I last heard from @SergiyVSergiyenko, UI does not use neither `declaredLostDate` nor `claimedReturnedDate`. They use metadata.updatedDate from the loan history record.

It seems that we can omit this property.

@SergiyVSergiyenko, @zburke, @mkuklis am I correct understand?

Reply:
@bohdan-suprun, this is not entirely true. To display the 'Claimed Return Date' value, we use `loan.claimedReturnedDate` property. And for the 'Declared Lost Date' value - really metadata.updatedDate (probably this will need to be fixed in the future).

Reply:
Aha, ok, thanks for clarifying this. We need the property then.